### DISCUSSION TITLE
The capacity  from output of zpool list have a huge difference with zfs list.

### DISCUSSION THREAD

**Discussion by guanfeix** [openzfs/zfs#14021] [Q&A]
![image](https://user-images.githubusercontent.com/34960209/195241185-8aa9d267-b120-419c-8e41-7fe0d011946f.png)
I build some thick-volume from one pool.But the capacity was far more beyond from my expectency.
I read the document of openzfs. Even though the method between the "zfs list"  and "zpool list",likes  slop space.But it doesn't fit to common sense. It really confussing me a lot.

**Comment by rincebrain** [comment]
`zfs list` will include things like if you have reservations that stop you from using that space for other things, like zvols default to being created with, or estimated space remaining after raidz overhead, while the "ALLOC" column of zpool list tells you how much space actually is allocated to data really written, would be my terse summary.

So in a simple example, if you have those zvols referencing, say, 750G each, but the volumes are sized 5.2T each, you'd end up with like 20.8T reserved and unavailable for anything else, but `zpool list` would say only 3T ALLOC.

**Reply by guanfeix** [reply]
tks, it' s quite clearly for me.
